ghzdac_cal/GHz_DAC_calibrate.py

```diff
@@ -120,8 +123,10 @@ def calibrate_pulse(cxn, fpga_name, dc_scope = 'infiniium'):
         board_type = 'dc'
     
     if board_type == 'ac':
-        zero_a, zero_b = zero_fixed_carrier(fpga_name)
-        dataset = calibrate_ac_pulse(fpga_name, zero_a, zero_b)
+        use_switch = reg.get(keys.SWITCHUSE)
```

Comment:
It would be good to set registry key to the default value in case it does not exist:
```python
use_switch = reg.get(keys.SWITCHUSE, 'b', True, True)
```
The third argument `True` means "set if the key does not exist" and the fourth argument is the default value to set, which just happens to be the same as the third arg in this case.